tox.ini

```diff
@@ -118,37 +118,48 @@ commands_pre =
   py3{6,7,8,9}: python -m pip install -U pip setuptools wheel
 ; Install common packages for all the tests. These are not needed in all the
 ; cases but it saves a lot of boilerplate in this file.
-  test: pip install {toxinidir}/opentelemetry-api {toxinidir}/opentelemetry-semantic-conventions {toxinidir}/opentelemetry-sdk {toxinidir}/tests/util
+  test: pip install {toxinidir}/opentelemetry-api {toxinidir}/opentelemetry-semantic-conventions {toxinidir}/opentelemetry-python-contrib/opentelemetry-instrumentation {toxinidir}/opentelemetry-sdk {toxinidir}/tests/util
+
+  test-core-sdk: pip install {toxinidir}/opentelemetry-python-contrib/opentelemetry-instrumentation
+  test-core-opentracing-shim: pip install {toxinidir}/opentelemetry-python-contrib/opentelemetry-instrumentation
 
   test-core-proto: pip install {toxinidir}/opentelemetry-proto
   distro: pip install {toxinidir}/opentelemetry-python-contrib/opentelemetry-instrumentation {toxinidir}/opentelemetry-distro
 
   getting-started: pip install -e {toxinidir}/opentelemetry-python-contrib/opentelemetry-instrumentation -e {toxinidir}/opentelemetry-python-contrib/instrumentation/opentelemetry-instrumentation-requests {toxinidir}/opentelemetry-python-contrib/util/opentelemetry-util-http -e {toxinidir}/opentelemetry-python-contrib/instrumentation/opentelemetry-instrumentation-wsgi -e {toxinidir}/opentelemetry-python-contrib/instrumentation/opentelemetry-instrumentation-flask
 
-  opencensus: pip install {toxinidir}/exporter/opentelemetry-exporter-opencensus
+  opencensus: pip install {toxinidir}/exporter/opentelemetry-exporter-opencensus {toxinidir}/opentelemetry-python-contrib/opentelemetry-instrumentation
```

Comment:
Do we still need to install this for the exporter environments if we are already doing it in `test`? (line 121)